src/Compiler/Checking/CheckFormatStrings.fs

```diff
@@ -16,25 +16,25 @@ open FSharp.Compiler.TcGlobals
 
 type FormatItem = Simple of TType | FuncAndVal 
 
-let copyAndFixupFormatTypar m tp = 
-    let _,_,tinst = FreshenAndFixupTypars m TyparRigidity.Flexible [] [] [tp]
+let copyAndFixupFormatTypar g m tp = 
```

Comment:
Anything longer than "g" will help :)

Reply:
`g` is a well-known from `TcGlobals, you can find others here: https://github.com/dotnet/fsharp/blob/main/docs/coding-standards.md